evals/eval.py

```diff
@@ -162,7 +163,31 @@ def worker_thread(args):
     def get_samples(self):
         if self.samples_jsonl is None:
             raise ValueError(
-                    "To use `get_samples`, you must provide a `samples_jsonl` path."
-                    "Got `None`.")
+                "To use `get_samples`, you must provide a `samples_jsonl` path." "Got `None`."
+            )
+
+        if os.path.exists(self.samples_jsonl):
+            with open(self.samples_jsonl, "r") as f:
+                first_line = f.readline()
+            if "version https://git-lfs.github.com/spec/v1" in first_line:
+                git_lfs_installed = False
+                try:
+                    subprocess.run(["git-lfs", "--version"], check=True, capture_output=True)
+                    git_lfs_installed = True
+                except subprocess.CalledProcessError:
+                    pass
+
+                if not git_lfs_installed:
+                    try:
+                        subprocess.run(["git", "lfs", "install"], check=True, capture_output=True)
+                    except subprocess.CalledProcessError as e:
+                        raise RuntimeError(
+                            "Failed to install git-lfs. Please install it manually."
```

Comment:
Was it `pip install git-lfs` to install this manually? Let's give a concrete step if the installation fails